crates/biome_css_analyze/src/keywords.rs

```diff
@@ -855,6 +855,59 @@ pub const OTHER_PSEUDO_ELEMENTS: [&str; 18] = [
     "view-transition-old",
 ];
 
+pub const MEDIA_FEATURE_NAMES: [&str; 50] = [
+    "device-aspect-ratio",
+    "device-height",
+    "device-width",
+    "aspect-ratio",
+    "color",
+    "color-index",
+    "height",
+    "monochrome",
+    "resolution",
+    "width",
+    "min-device-aspect-ratio",
+    "min-device-height",
+    "min-device-width",
+    "min-aspect-ratio",
+    "min-color",
+    "min-color-index",
+    "min-height",
+    "min-monochrome",
+    "min-resolution",
+    "min-width",
+    "max-device-aspect-ratio",
+    "max-device-height",
+    "max-device-width",
+    "max-aspect-ratio",
+    "max-color",
+    "max-color-index",
+    "max-height",
+    "max-monochrome",
+    "max-resolution",
+    "max-width",
+    "any-hover",
+    "any-pointer",
+    "color-gamut",
+    "display-mode",
+    "dynamic-range",
+    "forced-colors",
+    "grid",
+    "hover",
+    "inverted-colors",
+    "orientation",
+    "overflow-block",
+    "overflow-inline",
+    "pointer",
+    "prefers-color-scheme",
+    "prefers-contrast",
+    "prefers-reduced-motion",
+    "prefers-reduded-transparency",
+    "scripting",
```

Comment:
`"scan"` seems to be missing.

https://developer.mozilla.org/en-US/docs/Web/CSS/@media/scan